public-ui/src/app/views/breadcrumb/component.html

```diff
@@ -0,0 +1,15 @@
+<div class="breadcrumb-container">
+  <div class="breadcrumb">
+    <a href="https://www.staging.researchallofus.org/">Home</a>
+  </div>
+  <div class="breadcrumb">
+    <a href="https://www.staging.researchallofus.org/data-research-tools/">Data & Reasearch Tools</a>
+  </div>
+  <div class="breadcrumb">
+    <a href="https://www.staging.researchallofus.org/data-research-tools/data-browser/">About The Databrowser</a>
+  </div>
```

Comment:
Ideally these wouldn't be hardcoded, but looks like they're pointing to outside data browser so not sure what else would work. Maybe we could use a feature flag to point at the same environment you're currently using? Here's a [doc](https://docs.google.com/document/d/133JzAcrIf_ZKFqzL2jvVaBbxb69NMMSjs7iTuj8xW0A/edit#heading=h.pdgebek0f14m) CH created if you want to look into it.

Reply:
Also, I haven't seen the mocks for this, but the breadcrumbs are styled differently than the research hub's:

Reasearch Hub
![screen shot 2019-02-28 at 12 45 59 pm](https://user-images.githubusercontent.com/40036095/53590314-1a3e6100-3b57-11e9-9e3b-d83bcde71366.png)

Data Browser
![screen shot 2019-02-28 at 12 46 13 pm](https://user-images.githubusercontent.com/40036095/53590347-27f3e680-3b57-11e9-8c02-e11372757ff2.png)

Not sure if those need to be consistent or not.

Reply:
Sorry, I think having an environment variable with the url for is fine instead of following the feature flag doc. Looks like each environment file already has urls for workbench and data browser for each environment ([workbench test url](https://github.com/all-of-us/data-browser/blob/0b8c8fed94f7678ebc2a19fc85708297ed07757e/public-ui/src/environments/environment.staging.ts#L4)), maybe can add variables for research hub sites also.

Reply:
![image](https://user-images.githubusercontent.com/860209/53668558-c5bfe200-3c39-11e9-9a11-b8544496c4df.png)